references/bibtex/autotyp.bib

```diff
@@ -9273,13 +9273,21 @@ @incollection{2260
     lgcode = {Hawaiian Creole (autotyp: 1598 hwc hawa1247) = Hawai'i Creole English [hwc]}
 }
 @incollection{2261,
+    author = {Reynolds, Susan Bauder},
     editor = {Rickford, J. R. and Romaine, S.},
-    title = {Mutual Intellegibility?: Comprehension Problems between American Standard English and Hawai’i Creole English in Hawai’i’s Public Schools},
-    booktitle = {Creole Genesis, Attitudes and Discourse},
+  editor    = {Rickford, John R. and Romaine, Suzanne},
+  title     = {Mutual Intelligibility?},
+  booktitle = {Creole Genesis, Attitudes and Discourse. Studies celebrating Charlene J. Sato},
+    pages = {303-322},
+  inlg      = {English [eng]},
     year = {1999},
+    lgcode = {Hawaiian Creole (autotyp: 1598 hwc hawa1247) = Hawai'i Creole English [hwc]}
```

Comment:
This line is missing a comma at the end. Please run `glottolog check` before submitting a PR.